k8srm-prototype/pkg/api/claim_types.go

```diff
@@ -126,42 +74,95 @@ type DeviceClaim struct {
 // DeviceClaimSpec details the requirements that devices chosen
 // to satisfy this claim must meet.
 type DeviceClaimSpec struct {
-	// DeviceClass is the name of the DeviceClass containing the basic information
-	// about the device being requested.
+	// MatchAttributes allows specifying a constraint that will apply
+	// across all of the claims. For example, if you specified "numa", then
+	// this overall claim could only be successfully fulfilled if all of
+	// the included claims could be fulfilled by pools with the same "numa"
```

Comment:
naive question, where are these attributes coming from? are driver specific? 
how will a user know about those are valid or supported in the claim?

Reply:
Not naive. This is a good question. I think we will define some set of well-known attributes (e.g., vendor, model, numa, pcie-root, etc.), but otherwise they will be driver-specific.